vivisect/analysis/__init__.py

```diff
@@ -167,33 +176,63 @@ def addAnalysisModules(vw):
 
         vw.addAnalysisModule("vivisect.analysis.generic.entrypoints")
         vw.addAnalysisModule("vivisect.analysis.generic.funcentries")
-        vw.addAnalysisModule("vivisect.analysis.generic.relocations")
+        #vw.addAnalysisModule("vivisect.analysis.generic.relocations")
```

Comment:
blobs don't have relocations...  this should actually simply be removed.

Reply:
and findPointers really wants more enhancements (like checks if there's memory mapped at 0....), so for now, i disabled it for blob.

Reply:
since this is a architecture-agnostic issue, i'll change the pointertable one back, but since blobs have nor relocations, i'll delete that one altogether.